pkg/services/rpcsrv/subscription_test.go

```diff
@@ -376,7 +395,16 @@ func TestFilteredSubscriptions(t *testing.T) {
 					}
 					continue
 				}
-				this.check(t, resp)
+				if this.shouldCheck {
+					checked = true
+					this.check(t, resp)
+				} else {
+					t.Fatalf("msg: %s", resp.EventID())
```

Comment:
@AliceInHunterland, oh-oh, by `msg` from https://github.com/nspcc-dev/neo-go/pull/3715#discussion_r1879594528 I meant your error message, I just was too lazy to copy the whole message to the comment. Get the old message back.